DFC.App.ExploreCareers.Data/Contracts/IContentCacheService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DFC.App.ExploreCareers.Data.Contracts
+{
+    public interface IContentCacheService
```

Comment:
You do not  need to define this interface yourself as it is defined in the content nuget